tests/console/firewall_enabled.pm

```diff
@@ -15,10 +15,10 @@
 use base 'opensusebasetest';
 use strict;
 use testapi;
-use version_utils qw(is_jeos is_leap is_sle leap_version_at_least sle_version_at_least);
+use version_utils qw(is_jeos is_leap is_sle is_tumbleweed leap_version_at_least sle_version_at_least);
 
 sub run {
-    if ((is_sle && sle_version_at_least('15')) || (is_leap && leap_version_at_least('15.0'))) {
+    if ((is_sle && sle_version_at_least('15')) || (is_leap && leap_version_at_least('15.0')) || is_tumbleweed) {
```

Comment:
why not use is_storage_ng ? This should cover the differences for all systems

Reply:
tying firewalld to storage_ng? sounds less than intuitive

Reply:
Youre right, those are independent even though every of the mentioned systems are using it.
Then lgtm

Reply:
Can we please turn that around and still tumbleweed as the default? I.e. we assume now by default that the following if is true and we just handle the exceptions

Reply:
I doubt you mean to change this to ```if (!(…))``` and invert the commands between the if and else block.

But there is no better way to find stuff < sle15 and < leap15, or I missed to find it